# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/webkitgtk/hotfix-boolForPrinting.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- webkitgtk-2.36.0/Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp.vanilla	2022-03-28 20:05:10.066392911 +0200
+++ webkitgtk-2.36.0/Source/WebCore/platform/graphics/gstreamer/GStreamerCommon.cpp	2022-03-28 20:05:22.019393528 +0200
@@ -521,7 +521,7 @@
         gst_bus_disable_sync_message_emission(bus.get());
         GstState currentState;
         auto result = gst_element_get_state(pipeline, &currentState, nullptr, 0);
-        GST_DEBUG_OBJECT(pipeline, "Task finished, result: %s, target state reached: %s", gst_element_state_change_return_get_name(result), boolForPrinting(currentState == targetState));
+        //GST_DEBUG_OBJECT(pipeline, "Task finished, result: %s, target state reached: %s", gst_element_state_change_return_get_name(result), boolForPrinting(currentState == targetState));
     });
 
     result = gst_element_set_state(pipeline, targetState);
